### DISCUSSION TITLE
cannot write to /storage while building yamcs

### DISCUSSION THREAD

**Discussion by a-laborie** [yamcs/yamcs#657] [Q&A] [answered]
Hello everyone.
I have a problem while building the web interface as it is explained in README.md. When I run the bash run-example.sh with the simulation arg, I get the following message:
[INFO] Creating configuration at /home/alex/git/y/yamcs/examples/simulation/target/yamcs 14:08:17.912 _global [1] YamcsServer Failure while attempting to validate configuration: java.nio.file.AccessDeniedException: /storage
I tried to manually create the storage/yamcs-data manually and to give it all writting permission, but it did not changed anything. As a consequence, I cannot access http://localhost:8090/.
Is it a known problem or am I doing something wrong?
PS: I work on a linux machine.
Thank you
Alexandre Laborie

**Comment by xpromache** [comment, marked as answer]
There must be something with permissions, I cannot see anything else.

Can you run:
 touch /storage/yamcs-data/test123

as the same user which runs yamcs?

Maybe SELinux interfering? You can disable it (you can google "disable
selinux")

**Reply by a-laborie** [reply]
Hi and thank you for your reply. This wasn't a real problem. I was just writing on the wrong place...
